### DISCUSSION TITLE
SESI-8 CUYANIMELIST

### DISCUSSION THREAD

**Discussion by muhammadraflyeka** [deaafrizal/forum#22] [Troubleshooting Room]
guys!! ada yang bisa bantu gua kaga? gua ada error di bagian sign with Github pas di klik muncul peringatan Try signing in with a different account. tolong solusinya kawan-kawan ku

**Comment by deaafrizal** [comment]
coba berikut bro:
1. clear browser cache
2. bikin oauth github baru
3. save ulang oauth yang lagi dipake di dashboardnya github
4. pastikan credential sama & callback sesuai
